libnopegl/src/node_textureview.c

```diff
@@ -29,7 +29,7 @@
 static const struct node_param textureview_params[] = {
     {"texture", NGLI_PARAM_TYPE_NODE, OFFSET(texture),
                 .flags = NGLI_PARAM_FLAG_NON_NULL,
-                .node_types=(const uint32_t[]){NGL_NODE_TEXTURE2D, NGL_NODE_TEXTURECUBE, NGLI_NODE_NONE},
+                .node_types=(const uint32_t[]){NGL_NODE_TEXTURE2D, NGL_NODE_TEXTURECUBE, NGL_NODE_TEXTURE3D, NGLI_NODE_NONE},
```

Comment:
nit: move texture3d straight after texture2d (alphabetical); don't forget to regen doc/specs

Reply:
Done.